### PR TITLE
PE-915: Add missing fields and unit tests

### PR DESCRIPTION
**Pull request by mydagobah**
User story: <https://sailthru.atlassian.net/browse/PE-831>
Subtask: <https://sailthru.atlassian.net/browse/PE-915>

### CODE REVIEW

File: src/main/com/sailthru/client/params/Content.java [hunk 1]
```diff
@@ -27,6 +38,11 @@ public ApiAction getApiCall() {
 
     public static enum ContentSepecialVar {PRICE, DESCRIPTION, BRAND};
```
Comment (Dude-X) [inline src/main/com/sailthru/client/params/Content.java]: Can we fix this misspelling despite being outside of the scope of this ticket?

Comment (MerelyAPseudonym) [inline src/main/com/sailthru/client/params/Content.java]: **I'd** like that too...

**Comment by MerelyAPseudonym** [comment]
Assuming my concerns about the type changes of the output in https://github.com/sailthru/sailthru-java-client/commit/4a70470bf9e178a0a9a09806f380d5caafa004ee are addressed, this PR looks good to me.

**Comment by melcheng** [comment]
Let's also update the docs link in the POM file while we're at it to the new site:
http://getstarted.sailthru.com/

File: src/main/com/sailthru/client/params/Content.java (hunk 1 above)
Comment (mydagobah) [inline src/main/com/sailthru/client/params/Content.java]: good catch. fixed.

**Pull request merged**